i18n/example_test.go

```diff
@@ -22,6 +23,7 @@ func Example() {
 	fmt.Println(T("your_unread_email_count", 1))
 	fmt.Println(T("your_unread_email_count", 2))
 	fmt.Println(T("my_height_in_meters", "1.7"))
+	fmt.Println(T("my_height_in_meters", countStruct))
```

Comment:
Instead of this test, can you please test 0,1,2 to make sure that pluralization is working correctly? Also it would be good to test both structs and maps.

```go
type Count struct { Count int }
fmt.Println(T("your_unread_email_count", Count{0}))
fmt.Println(T("your_unread_email_count", Count{1}))
fmt.Println(T("your_unread_email_count", Count{2}))

fmt.Println(T("your_unread_email_count", map[string]string{"Count": 0}))
fmt.Println(T("your_unread_email_count", map[string]string{"Count": 1}))
fmt.Println(T("your_unread_email_count", map[string]string{"Count": 2}))
```